examples/test/canvas/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Video</title>
+    <meta name="description" content="Video - A-Frame">
+    <script src="../../../dist/aframe.js"></script>
+  </head>
+  <body>
+    <script type="text/javascript">
+      window.onload = function(){
```

Comment:
Extra credit to use the component system to hook into A-Frame's requestAnimationFrame and for reusability.

```js
AFRAME.registerComponent('update-canvas', {
  schema: {default: ''},

  init: function () {
    this.canvas = document.getElementById(this.data);
    this.ctx = canvas.getContext('2d');
  },

  tick: function (t) {
    var hue = t / 10;
    // ...
  }
});
```

```html
<a-scene update-canvas="helloworld-canvas">
</a-scene>
```